config/vufind/Demo.ini

```
@@ -30,6 +30,10 @@ ILLRequests = true
 ; "bound-with" information. (Default = 25% chance)
 ;boundWithProbability = 25
 
+; Whether or not you don't want random holdings generated
+; (Default : false, meaning random holdings will be generated)
+;NoRandomHoldings = true
```

Maybe it would be more clear to reverse this:

```suggestion
; When set to true (the default), the driver will generate random holdings for
; all records; even items with [StaticHoldings] will have additional random data
; added. If set to false, then only holdings in [StaticHoldings] will be used.
;generateRandomHoldings = true
```

Also, once it is finalized, maybe it would make sense to move these minor Demo driver improvements to a separate PR so we can merge them early and reduce the remaining complexity of this PR.